config/application.rb

```diff
@@ -22,8 +22,14 @@ class Application < Rails::Application
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
     config.active_record.raise_in_transactional_callbacks = true
-    
+
     # Gzip files when possible
     config.middleware.use Rack::Deflater
+
+    #HCaseflow application urls
+    config.app_urls = {"Caseflow" => "https://caseflow.ds.va.gov", "Efolder" => "https://efolder.cf.ds.va.gov"}
```

Comment:
Also, hard coding URLs like this won't work in lower environments. For example, in dev, we'd want the redirect to go here: https://dsva-appeals-certification-dev-1895622301.us-gov-west-1.elb.amazonaws.com, not to https://certification.cf.ds.va.gov